vuln/npm/413.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": 413,
+  "title": "Path Traversal",
+  "overview": "[mcstatic] Path Traversal allows to read content of arbitrary files",
+  "created_at": "2018-02-06",
+  "updated_at": "2018-04-24",
+  "publish_date": "2018-04-24",
+  "author": "bl4de (https://twitter.com/_bl4de)",
+  "module_name": "mcstatic",
+  "cves": [
+    "CVE-2018-3730"
+  ],
+  "vulnerable_versions": "0.0.20",
```

Comment:
This particular version only? Is `0.0.19` not vulnerable?

Reply:
Correct!